third_party/wabt/include/wabt/interp/jit/jit.h

```diff
@@ -22,6 +22,11 @@
 namespace wabt {
 namespace interp {
 
+using ValueType = wabt::Type;
```

Comment:
is `class ValueType;` forward declaration does not work?

Reply:
No, the compiler gives an error:
`error: using typedef-name ‘using ValueType = class wabt::Type’ after ‘class’`

because `ValuType` has a previous declaration using `using` in `istram.h` and in `interp.h`